control_panel_api/tests/test_models.py

```diff
@@ -40,6 +42,20 @@ def test_helm_create_user(self):
         helm.config_user.assert_called_with(username)
         helm.init_user.assert_called_with(username, email, name)
 
+    def test_helm_delete_user(self):
+        username = 'foo'
+        email = 'bar@baz.com'
+        name = 'bat'
+        user = User.objects.create(
```

Comment:
In this test you don't need to persist the user strictly speaking.
You could just build a user object with `User(username=username)` and save a trip to the DB :) (which make this test a bit faster).

Reply:
👍